servers/src/main/java/tachyon/StorageTierAssoc.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.ImmutableBiMap;
+
+import tachyon.conf.TachyonConf;
+
+/**
+ * Creates a two-way mapping between StorageTier aliases and ordinal numbers from the given
+ * TachyonConf. It can be used for both worker and master storage tier mapping. This class is thread
+ * safe.
+ */
+public class StorageTierAssoc {
+  /*
+   * An immutable bi-directional mapping between storage level aliases and their ordinals. Immutable
+   * maps are thread safe.
+   */
+  private final ImmutableBiMap<String, Integer> mAliasToOrdinal;
+
+  /*
+   * Constructs a new instance using the given TachyonConf object. The mapping cannot be modified
+   * after creation.
+   *
+   * @param conf the TachyonConf to build the mapping from
+   *
+   * @param levelsProperty the property in the conf that specifies how many levels there are
+   *
+   * @param aliasFormat the format for the conf that identifies the alias for each level
+   */
+  public StorageTierAssoc(TachyonConf conf, String levelsProperty, String aliasFormat) {
```

Comment:
I think you can make this a base class and make a wrapper for the master/worker ones?